templates/admin/job_template/index.html.ep

```diff
@@ -76,6 +87,66 @@
             </select>
 
         </div>
+        <form action="#" id="editor-form" class="form-horizontal" style="display: none;" onsubmit="return submitTemplateEditor();"
+          data-put-url="<%= url_for('apiv1_job_templates_schedules' => (id => $group->id)) %>">
+            <div class="form-group row">
+                <label for="editor-template" class="col-sm-4 control-label" data-toggle="tooltip" title="Changes in the YAML will be reflected in the database">
+                    <span id="editor-yaml-guide">
+                        <div>Define test suites per arch/ medium:</div>
+                        <code>
+<b>'My Group'</b>:
+  architectures:
+    <b>ARCH</b>:
+      <i>medium</i>-<b>ARCH</b>:
+      - test_suite
+      - test_suite:
+        machine: <b>MACHINE</b>
+        priority: <b>PRIORITY</b>
+  defaults:
+    <b>ARCH</b>:
+      machine: <b>MACHINE</b>
+      priority: <b>PRIORITY</b>
+  products:
+    <i>medium</i>-<b>ARCH</b>:
+      distribution: <b>DISTRIBUTION</b>
+      flavor: <b>FLAVOR</b>
+      version: <b>VERSION</b>
+                        </code>
+                        <div>Re-use test suites with YAML aliases:</div>
+                        <code>
+architectures:
+  <b>i586</b>:
+    <i>medium</i>-<b>i586</b>: <b>&amp;tests</b>
+    - foo
+    - bar:
+      machine: <b>64bit</b>
+      priority: <b>70</b>
+  <b>x86_64</b>:
+    <i>medium</i>-<b>x86_64</b>:
+      <b>*tests</b>
+                        </code>
+                    </span>
+                </label>
+                <div class="col-sm-8">
+                    <textarea class="form-control" id="editor-template" name="template" rows="30" readOnly="true"></textarea>
```

Comment:
To set the height of the text area in accordance with the example you could add `$('#editor-template').height($('#editor-form label').outerHeight());` somewhere in the JavaScript code. (Not sure how to do it in CSS.)